src/api/Controllers/CasingController.cs

```diff
@@ -67,13 +65,59 @@ public async Task<ActionResult<Casing>> GetByIdAsync(int id)
 
     /// <inheritdoc />
     [Authorize(Policy = PolicyNames.Viewer)]
-    public override Task<ActionResult<Casing>> CreateAsync(Casing entity)
-        => base.CreateAsync(entity);
+    public override async Task<ActionResult<Casing>> CreateAsync(Casing entity)
+    {
+        if (entity == null) return BadRequest();
+
+        try
+        {
+            if (!(entity.CasingElements?.Count > 0))
```

Comment:
PP: Die `if`-Anweisung umkehren, so dass sie einfacher lesbar ist:
```suggestion
            if (entity.CasingElements?.Count == 0)
```

Reply:
@tschumpr Alles klar, dann stimmt das so. Müsste dann das andere `if`-Statement in `EditAsync` auch angepasst werden?